security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -13,6 +13,52 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-55"]
+== OpenShift Compliance Operator 0.1.55
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.55:
+
+* link:https://access.redhat.com/errata/RHBA-2022:6657[RHBA-2022:6657 - OpenShift Compliance Operator bug fix update]
+
+[id="compliance-operator-0-1-55-new-features-and-enhancements"]
+=== New features and enhancements
+
+* `KubeletConfig` now checks the default configuration of the `KubeletConfig`. The configuration files are aggregated from all nodes into a single location. See xref:../../security/compliance_operator/compliance-operator-remediation.adoc#compliance-evaluate-kubeletconfig-rules[Evaluating `KubeletConfig` rules against default configuration values].
```

Comment:
nit: into a single location per node pool?

Reply:
we should probably explicitly say that the kubelet checks changed their type from `Node` to `Platform` as part of this change. Even in internal testing, there were people who were running only node checks and thought that the kubelet checks have disappeared, see https://issues.redhat.com/browse/CMP-1591